src/services/appointment.service.ts

```diff
@@ -83,12 +84,17 @@ export default class AppointmentService implements OnModuleInit {
       customer,
       treatments,
     });
-
-    let savdAppt;
+    const eventId = await this.googleCalendarService.createEvent(appointment);
+    appointment.eventId = eventId;
+    let savedAppointment;
 
     try {
-      savdAppt = await this.appointmentRepository.save(appointment);
+      savedAppointment = await this.appointmentRepository.save(appointment);
```

Comment:
💭 I think the send appointment confirmation should go inside the try block as well as the schedule cancellation